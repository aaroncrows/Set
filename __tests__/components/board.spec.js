import React from 'react'
import { create } from 'react-test-renderer'
import { Provider } from 'react-redux'
import mockStore from '../lib/mockStore'

import Board from '../../src/components/board'

let testState
beforeEach(() => {
  testState = {
    selectedCards: [],
    board: [[{shape: 'testShape', pattern: 'testPattern', color: 'testColor', count: 'testCount'}]]
  }
})

test('should render correctly', () => {
  const testRender = create(<Provider store={mockStore(testState)}><Board /></Provider>)

  expect(testRender).toMatchSnapshot()
})
