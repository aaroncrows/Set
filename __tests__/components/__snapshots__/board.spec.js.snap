// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`container render correctly 1`] = `
<ul
  className="board"
>
  <ul>
    <li
      className="card"
      onClick={[Function]}
    >
      <span
        className="icons-one color-one icon-leaf"
      />
      <span
        className="pattern"
      >
        solid
      </span>
    </li>
  </ul>
</ul>
`;
